Add support for schedulerName field in the Pod template (#2114)
strimzi/strimzi-kafka-operator: documentation/book/con-customizing-pods.adoc

|Configures the name of the Priority Class which will be used for given a Pod.
For more information about Priority Classes, see {K8sPriorityClass}.

|`schedulerName`
| The name of the scheduler used to dispatch this `Pod`.

> If not specified, the default scheduler will be used.

Reviewer: ```suggestion
If not specified, the default scheduler is used.
```
